pennylane_lightning/_serialize.py

```diff
@@ -17,13 +17,7 @@
 from typing import List, Tuple
 
 import numpy as np
-from pennylane import (
-    BasisState,
-    Hadamard,
-    Projector,
-    QubitStateVector,
-    Rot,
-)
+from pennylane import BasisState, Hadamard, Projector, QubitStateVector, Rot
```

Comment:
I also prefer the original format, but black (the latest version) reformatted like this...

Reply:
Huh, I guess we must do what black tells us :shrug:

Reply:
I found that I was using Python3.7 (due to some misconfiguration...). Thus I was using an old version (instead of the latest) version of black. After I change to Python 3.8, black passed, so I rollbacked some code blocks.